src/and.js

```diff
@@ -1,28 +1,24 @@
 var _curry2 = require('./internal/_curry2');
-var _hasMethod = require('./internal/_hasMethod');
 
 
 /**
- * A function that returns the first argument if it's falsy otherwise the second
- * argument. Note that this is NOT short-circuited, meaning that if expressions
- * are passed they are both evaluated.
- *
- * Dispatches to the `and` method of the first argument if applicable.
+ * Returns `true` if both arguments are `true` and otherwise `false`.
```

Comment:
I suggest this tweak:

```diff
- * Returns `true` if both arguments are `true` and otherwise `false`.
+ * Returns `true` if both arguments are `true`; `false` otherwise.
```